packages/cli/src/controllers/password-reset.controller.ts

```diff
@@ -103,6 +104,12 @@ export class PasswordResetController {
 			throw new UnprocessableRequestError('forgotPassword.ldapUserPasswordResetUnavailable');
 		}
 
+		const oidcIdentity = user.authIdentities?.find((i) => i.providerType === 'oidc');
+
+		if (this.licenseState.isOidcLicensed() && oidcIdentity) {
```

Comment:
I think we should have the same rule as SAML, no ? Especially handling the allowSSOManualLogin thingy